Description: Solve alignment issues on i386.
 See the patch below for more information.
Origin: http://launchpadlibrarian.net/379337785/qtwebengine-opensource-src_5.11.1+dfsg-2ubuntu3_5.11.1+dfsg-2ubuntu4.diff.gz

--- a/src/3rdparty/chromium/mojo/public/c/system/macros.h.gcc8-alignof	2018-05-15 14:58:46.448912634 -0400
+++ b/src/3rdparty/chromium/mojo/public/c/system/macros.h	2018-05-15 14:58:52.041784613 -0400
@@ -18,7 +18,13 @@
 #endif

 // Like the C++11 |alignof| operator.
-#if __cplusplus >= 201103L
+#if defined(__GNUC__) && __GNUC__ >= 8
+// GCC 8 has changed the alignof operator to return the minimal alignment
+// required by the target ABI, instead of the preferred alignment.
+// This means that on 32-bit x86, it will return 4 instead of 8.
+// Use __alignof__ instead to avoid this.
+#define MOJO_ALIGNOF(type) __alignof__(type)
+#elif __cplusplus >= 201103L
 #define MOJO_ALIGNOF(type) alignof(type)
 #elif defined(__GNUC__)
 #define MOJO_ALIGNOF(type) __alignof__(type)
